source/game/field/obj/objects/Mdush.hh

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "game/field/obj/GeoObject.hh"
+#include <egg/math/Matrix.hh>
+
+namespace Field
+{
+
+class Mdush : public GeoObjectKCL {
+public:
+    Mdush(System::MapdataGeoObj* pMapDataGeoObj) : GeoObjectKCL(pMapDataGeoObj) {}
```

Comment:
Unless @vabold Feels differently, I believe the definition must go in the source.